test/vignette/media_types_test.clj

```diff
@@ -25,12 +25,11 @@
 
 (def filled-map (assoc latest-map :options {:fill "green"}))
 
-(def lang-map (assoc latest-map :options {:lang "es"}))
+(def lang-map (assoc latest-map :options {:path-prefix "es"}))
 
-(def prefix-path-map (assoc latest-map :options {:lang "de"
-                                                 :path-prefix "pokemanshop/zh"}))
+(def prefix-path-map (assoc latest-map :options {:path-prefix "pokemanshop/zh/de"}))
 
-(def lang-original-map (assoc original-map :options {:lang "es"}))
+(def lang-original-map (assoc original-map :options {:path-prefix "es"}))
 
```

Comment:
Want to add a test for `original-path` that includes the zone to confirm that the correct path on disk is generated?